src/graphics/gfx_enter_screen.cpp

```diff
@@ -18,56 +18,133 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+#include <SDL2/SDL_timer.h>
+
 #include "../globals.h"
 #include "../graphics.h"
 #include "../player.h"
+#include "../frame_timer.h"
+#include "../screen_fader.h"
+#include "pge_delay.h"
 
 
-void GameThing()
+static void initPlayers(Player_t tempPlayer[2])
 {
-    int A = 0;
-//    int B = 0;
-    Player_t tempPlayer[3];
+    for(int A = 0; A < numPlayers; ++A)
```

Comment:
Dangerous, what if player used `supermario128` and numPlayers > maxLocalPlayers?

Reply:
Lemme try...

Reply:
But, you are absolutely right, the tempPlayer[] will result out of range

Reply:
So, this condition should be right:
```cpp
A < numPlayers && A < maxLocalPlayers
```